src/desktop/apps/article/templates/__tests__/meta.jest.js

```diff
@@ -12,7 +13,7 @@ const render = templateName => {
 
 describe("Meta template", () => {
   it("contains basic meta tags", () => {
-    const article = _.extend({}, fixtures.article, {
+    const article = _.extend({}, StandardArticle, {
```

Comment:
The fixture must be getting overridden in another test somewhere, these came up with an incorrect title until I swapped a different fixture in.